src/graph/DescribeSpaceExecutor.cpp

```diff
@@ -0,0 +1,59 @@
+/* Copyright (c) 2018 - present, VE Software Inc. All rights reserved
```

Comment:
Please update the copyright header.

Do the same to all of your pending PRs, if any new file were added.

Reply:
fixed